src/sim_recon/images/dataclasses.py

```diff
@@ -15,13 +16,20 @@
 logger = logging.getLogger(__name__)
 
 
+class ProcessingStatus(StrEnum):
+    UNSET = auto()
+    PENDING = auto()
+    FAILED = auto()
+    COMPLETE = auto()
+
+
 @dataclass(frozen=True, slots=True)
 class BoundMrc:
     array: NDArray[Any]
     mrc: Mrc
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=False, slots=True)
```

Comment:
```suggestion
@dataclass(slots=True)
```
to be consistent with L44